seatunnel-server/seatunnel-app/src/main/bin/seatunnel-backend-daemon.sh

```diff
@@ -74,7 +74,7 @@ stop() {
 
 #status
 status() {
-  pid=$(jcmd | grep -i 'seatunnel-app-.*jar' | grep -v grep | awk '{print $1}')
+  pid=$(jcmd | grep -i 'org.apache.seatunnel.app.SeatunnelApplication' | grep -v grep | awk '{print $1}')
```

Comment:
Nice catch, @xiaochen-zhou. 
Could you also make additional modifications by changing seatunnel to seatunnel-web?
like below
```
if [ -n "$pid" ]; then
    echo "seatunnel-web is running"
  else
    echo "seatunnel-web is not running"
  fi
```

Reply:
> Nice catch, @xiaochen-zhou. Could you also make additional modifications by changing seatunnel to seatunnel-web? like below
> 
> ```
> if [ -n "$pid" ]; then
>     echo "seatunnel-web is running"
>   else
>     echo "seatunnel-web is not running"
>   fi
> ```

done.